util.go

```diff
@@ -137,3 +139,58 @@ func SetRound(block VerifiedBlock, notarization *Notarization, fCert *Finalizati
 
 	return round
 }
+
+type oneTimeVerifier struct {
```

Comment:
I probably should add a UT only for this, and a UT that checks that we have verified two blocks of the same round but different digests